src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -115,11 +139,30 @@ public static void deleteTempTargetBQTable() throws IOException, InterruptedExce
     }
   }
 
+  @After(order = 1, value = "@BQ_SINK_TEST_BQMT")
+  public static void deleteTempTargetBQMTTable() throws IOException, InterruptedException {
+    try {
+      BigQueryClient.dropBqQuery(PluginPropertyUtils.pluginProp("bqmtCsvTargetTable1"));
```

Comment:
Use for loop over comma separated list of tables

Reply:
Added for loop over comma separated list of tables for all respective hooks.